app/models/agents/telegram_agent.rb

```diff
@@ -68,18 +84,23 @@ def telegram_bot_uri(method)
     end
 
     def receive_event(event)
-      TELEGRAM_ACTIONS.each do |field, method|
+      messages_send = TELEGRAM_ACTIONS.map do |field, method|
```

Comment:
Maybe you could say `messages_sent = TELEGRAM_ACTIONS.count do |field, method|` here and change the condition below from `if messages_send.empty?` to `if messages_sent.zero?`.

Reply:
Good idea